### PR TITLE
Converting README.rst to README.md and fixing pyproject.toml

### PR DESCRIPTION
**Pull request by mikhail-akimov**
Fixing Issue-6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Jinja2 extension to handle git-specific things
+==============================================
```
Comment (sobolevn) [inline README.md]: This should be `# Jinja2 extension to handle git-specific things`, without `===` part

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+[![image](https://travis-ci.org/sobolevn/jinja2-git.svg?branch=master)](https://travis-ci.org/sobolevn/jinja2-git)
```
Comment (sobolevn) [inline README.md]: Please, rename alts to match the domain: `travis` for travis, `coveralls` for coveralls, etc. Alts are `[image]` tags

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+[![image](https://badge.fury.io/py/jinja2-git.svg)](http://badge.fury.io/py/jinja2-git)
+[![image](https://img.shields.io/pypi/pyversions/jinja2-git.svg)](https://pypi.python.org/pypi/jinja2-git)
+
+Reasoning
+---------
+
+This plugin is used to render commit hash in `jinja2` templates. We are
+using it to render our template version in `cookicutter`:
+
+-   [wemake-django-template](https://github.com/wemake-services/wemake-django-template)
```
Comment (sobolevn) [inline README.md]: Extra whitespaces

File: README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+-   [wemake-vue-template](https://github.com/wemake-services/wemake-vue-template)
```
Comment (sobolevn) [inline README.md]: Extra whitespaces

File: README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Usage
```
Comment (sobolevn) [inline README.md]: Should be `## Usage`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+``` {.sourceCode .python}
+from jinja2 import Environment
+
+env = Environment(extensions=['jinja2_git.GitExtension'])
+template = env.from_string('Commit is: {% gitcommit %}')
+# => Commit is: c644682f4899d7e98147ce3a61a11bb13c52b3a0
+```
+
+Installation
```
Comment (sobolevn) [inline README.md]: Should be `## Installation`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+------------
+
+``` {.sourceCode .bash}
```
Comment (sobolevn) [inline README.md]: Should be ` ```bash `

**Comment by mikhail-akimov** [comment]
And some other mistakes was also resolved.

**Comment by sobolevn** [comment]
Nice!

**Pull request merged**